platforms/ios/demo/src/MapViewController.h

```diff
@@ -12,12 +12,16 @@
 
 - (void)mapView:(nonnull TGMapView *)mapView didLoadScene:(int)sceneID withError:(nullable NSError *)sceneError;
 - (void)mapViewDidCompleteLoading:(nonnull TGMapView *)mapView;
+- (void)mapViewRegionIsChanging:(TGMapView *)mapView;
```

Comment:
Nit: Unnecessary to define these in the header as its implied by the `TGMapViewDelegate` protocol conformance. It doesn't hurt anything but it's unnecessary.